assets/dev/js/editor/components/ai/components/prompt-action-selection.js

```diff
@@ -0,0 +1,50 @@
+import { MenuItem, Select, FormControl, InputLabel } from '@elementor/ui';
+
+const labelToDashCash = ( str ) => str.toLowerCase().replace( /\s/g, '-' );
```

Comment:
it will also replace new lines with dash, just making sure that you're ok with that

Reply:
Done.